Bottom: fc003681d141a917d0342feb3ef20229c85124fb
Top:    7a76969c15cec2e5bbb17ef234f067ffcc000890
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 23:37:16 +0100

Refresh of try-something-with-the-rtrees

---

diff --git a/src/polygon1.c b/src/polygon1.c
index b2a11cb..13f56b4 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -850,7 +850,7 @@ intersect (jmp_buf * jb, POLYAREA * b, POLYAREA * a, int add)
       sb.X2 = pa->xmax + 1;
       sb.Y2 = pa->ymax + 1;
 
-      r_search (b_contour_tree, &sb, NULL, contour_bounds_touch, &c_info);
+      r_search (b->contour_tree, &sb, NULL, contour_bounds_touch, &c_info);
     }
 
   return 0;
